components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -312,15 +315,15 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
             instancesWithReset.map((i) => {
                 const req = new SetTimeoutRequest();
                 req.setId(i.id);
-                req.setDuration(defaultTimeout);
+                req.setDuration(this.userService.workspaceTimeoutToDuration(defaultTimeout));
 
                 return client.setTimeout(ctx, req);
             }),
         );
 
         const req = new SetTimeoutRequest();
         req.setId(runningInstance.id);
-        req.setDuration(duration);
+        req.setDuration(this.userService.workspaceTimeoutToDuration(duration));
```

Comment:
Nit: Unfortunate variable naming 🙈 `somethingElseToDuration(duration)` ("...wait, isn't it already a duration?")